src/app/static/src/tests/mocks.ts

```diff
@@ -3,16 +3,24 @@ import {GitState} from "../js/store/git/git";
 import {RunReportRootState} from "../js/store/runReport/store";
 import {ActionContext} from "vuex";
 
-export const mockRunReportMetadata = (props: Partial<RunReportMetadata> = {}): RunReportMetadata => {
+export type RecursivePartial<T> = {
```

Comment:
Wow, nice!

Reply:
we had this in a prev project, webapps maybe? just forgot about it!